common/scaffolding/interactive-content-en.html

```diff
@@ -0,0 +1,365 @@
+Your browser does not support the video tag.
+&lt;/video&gt;</code>
+</pre>
+
+<h2>The <code>audio</code> element</h2>
+<audio controls>
+	<source src="https://www.archive.org/download/RideOfTheValkyries/ride_of_the_valkyries_2.mp3" type="audio/mp3"/>
+	<source src="https://www.archive.org/download/RideOfTheValkyries/ride_of_the_valkyries_2.ogg" type="audio/ogg"/>
+	Your browser does not support the audio tag.
+</audio>
+<pre>
+<code>&lt;audio controls&gt;
+&lt;source src="https://www.archive.org/download/RideOfTheValkyries/ride_of_the_valkyries_2.mp3" type="audio/mp3"/&gt;
+&lt;source src="https://www.archive.org/download/RideOfTheValkyries/ride_of_the_valkyries_2.ogg" type="audio/ogg"/&gt;
+Your browser does not support the audio tag.
+&lt;/audio&gt;</code>
+</pre>
+
+<h2>The <code>track</code> element</h2>
+<video controls>
+	<source src="https://wet-boew.github.io/wet-boew-attachments/videos/video2-en.webm" type="video/webm">
+	<source src="https://wet-boew.github.io/wet-boew-attachments/videos/video2-en.mp4" type="video/mp4">
+	<track src="demo/video2-captions-en.xml" kind="captions" data-type="application/ttml+xml" srclang="en" label="English">
+	Your browser does not support the video tag.
+</video>
+<pre>
+<code>&lt;video controls&gt;
+&lt;source src="https://wet-boew.github.io/wet-boew-attachments/videos/video2-en.webm" type="video/webm"&gt;
+&lt;source src="https://wet-boew.github.io/wet-boew-attachments/videos/video2-en.mp4" type="video/mp4"&gt;
+&lt;track src="demo/video2-captions-en.xml" kind="captions" data-type="application/ttml+xml" srclang="en" label="English"&gt;
+Your browser does not support the video tag.
+&lt;/video&gt;</code>
+</pre>
+
+<h2>The <code>map</code> and <code>area</code> elements</h2>
+
+<img src="img/planets.gif" width="145" height="126" alt="Planets"
+usemap="#planetmap">
+
+<map name="planetmap">
+<area shape="rect" coords="0,0,82,126" href="img/sun.gif" alt="Sun">
+<area shape="circle" coords="90,58,3" href="img/mercury.gif" alt="Mercury">
+<area shape="circle" coords="124,58,8" href="img/venus.gif" alt="Venus">
+</map>
+
+<pre>
+<code>&lt;img src="img/planets.gif" width="145" height="126" alt="Planets"
+usemap="#planetmap"&gt;
+
+&lt;map name="planetmap"&gt;
+&lt;area shape="rect" coords="0,0,82,126" href="img/sun.gif" alt="Sun"&gt;
+&lt;area shape="circle" coords="90,58,3" href="img/mercury.gif" alt="Mercury";
+&lt;area shape="circle" coords="124,58,8" href="img/venus.gif" alt="Venus";
+&lt;map name="planetmap"&gt;
+</code>
+</pre>
+
+<h2>The <code>MathML</code> element</h2>
+<h3>The quadratic formula</h3>
+<p>
+<math>
+<mi>x</mi>
+<mo>=</mo>
+<mfrac>
+	<mrow>
+	<mo form="prefix">−</mo> <mi>b</mi>
+	<mo>±</mo>
+	<msqrt>
+	<msup> <mi>b</mi> <mn>2</mn> </msup>
+	<mo>−</mo>
+	<mn>4</mn> <mo>⁢</mo> <mi>a</mi> <mo>⁢</mo> <mi>c</mi>
+	</msqrt>
+	</mrow>
+	<mrow>
+	<mn>2</mn> <mo>⁢</mo> <mi>a</mi>
+	</mrow>
+</mfrac>
+</math>
+</p>
+
```

Comment:
Can you add an informative image to show what the visual rendering are expected?

```
<h4>Expected rendering</h4>
<img src="img/mathml.png" alt="See the previous section 'The quadratic formula' for the image content">
```
mathml.png can be this:
![image](https://github.com/wet-boew/GCWeb/assets/1676030/1626fb47-4708-4b98-babe-729d875e6c15)

Reply:
Fixed in commit "Fixed errors and added images".